pkg/trace/info/stats.go

```diff
@@ -86,22 +89,26 @@ func (rs *ReceiverStats) Reset() {
 }
 
 // Strings gives a multi strings representation of the ReceiverStats struct.
-func (rs *ReceiverStats) Strings() []string {
+func (rs *ReceiverStats) Strings() (infoStrings []string, warnStrings []string) {
```

Comment:
`Strings` is such a confusing name, given that `String` is so commonly used in Go as `fmt.Stringer` implementations. Maybe `Report` would be a better name.

Reply:
Or even better, we could call it `Log` and have this very method call `log.Info` and `log.Warn`. It seems like it would be much cleaner. What do you think?

Reply:
Good point, that is simpler. Refactored https://github.com/DataDog/datadog-agent/commit/6f0b21852341dcde2b3d6e12998695af640e5d22